local-cluster/tests/common/mod.rs

```diff
@@ -79,7 +79,10 @@ pub fn restore_tower(tower_path: &Path, node_pubkey: &Pubkey) -> Option<Tower> {
 
 pub fn remove_tower(tower_path: &Path, node_pubkey: &Pubkey) {
     let file_tower_storage = FileTowerStorage::new(tower_path.to_path_buf());
-    fs::remove_file(file_tower_storage.filename(node_pubkey)).unwrap();
+    let filename = file_tower_storage.filename(node_pubkey);
```

Comment:
why make this and `wait_for_last_vote_in_tower_to_land_in_ledger` succeed when the tower is not present? seems like it silences programmer errors?

Reply:
1. For the `wait_for_last_vote_in_tower_to_land_in_ledger` the errors aren't silenced because empty towers return None which panic when unwrapped
2. For `remove_tower()` my test had a case where there's a race where the tower sometimes doesn't exist if the validator was killed before it could vote. I can make a separate function `remove_tower_if_exists`

Reply:
got it, ya would prefer to have `remove_tower_if_exists` just so people are aware of any races

Reply:
yup, added!